code_challenges.rb

```diff
@@ -2,36 +2,86 @@
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in an array of words and a letter and returns all the words that contain that particular letter.
+# I want to create a method. First thing you want to do when creating a method is using a def and and end.
+#  I attempted to look through the syllabus and bravo lecture notes to assist me with this problem but after an hour on this problem I needed to move on. I would like to review this problem so I can practice the correct approach to taking in an array of words and a letter and returns all the words that contain that letter.
 
-beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
-letter_o = 'o'
-# Expected output: ['coffee', 'soda water']
-letter_t = 't'
-# Expected output: ['tea', 'water', 'soda water']
 
+# beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+#  letter_o = 'o'
+#  def finder(beverages_array) 
+#     array.select do |value| 
+#         value.downcase.start_with? ("o")
```

Comment:
so here you are calling on the letter "o" in your code. What if I as a user wanted to sort by the letter "r" but I don't have access to your code base?  What changes would have to be made to your code to make it dynamic and not static?